xml/operating_system.xml

```diff
@@ -705,6 +715,7 @@
   <fingerprint pattern="^(?i:(?:Cisco\s?)?(ASA|Adaptive Security Appliance|IOS|IOS-XE|IOS-XR|NX-OS|PIX-OS|SAN-OS)\s?(?:Version (\S+))?)$">
     <description>Cisco</description>
     <example os.family="ASA" os.product="ASA">Cisco ASA</example>
+    <example os.family="ASA" os.product="ASA" os.version="1.0">Cisco ASA Version 1.0</example>
```

Comment:
From a quick look at [Cisco ASA New Features by Release](https://www.cisco.com/c/en/us/td/docs/security/asa/roadmap/asa_new_features.html) it appears the first release was `ASA 7.0(1)` on May 31, 2005.

Reply:
Fixed the regular expression and added a couple of examples